### PR TITLE
230 add search index

### LINKED ISSUE #230: Set up content query index for the search functionality

### PR DESCRIPTION
**Pull request by nc-andreashaller**
Fix #230 

Test URLs:
- Before: https://main--cn-website--netcentric.hlx.live/
- After: https://230-add-search-index--cn-website--netcentric.hlx.live/

### CODE REVIEW

File: helix-query.yaml [hunk 1]
```diff
@@ -70,6 +70,32 @@ indices:
...
         value: innerHTML(el)
+      lastModified:
+        select: none
+        value: parseTimestamp(headers["last-modified"], "ddd, DD MMM YYYY hh:mm:ss GMT")
+  search:
+    include:
+      - '/**'
+    exclude:
+      - '/de/**'
+      - '/de'
```
Comment (nc-andreashaller) [inline helix-query.yaml:81]: @jvaldas we could remove this to have all languages in one file, but I think to keep the file a little smaller it actually makes sense to separate it and create another index under /de/query-index-search.json later. The thing is you would need to have some logic around this in the lambda, but I assume we would need that logic anyway to separate the two languages into separate indexes. Wdyt?

Comment (AngelaFabregues) [reply helix-query.yaml:81]: Let’s keep the search solution decoupled from the website. All content extracted and made available, please.
Is there mandatory to use this new target name? I’m lazy. I don’t want to change all the specification files again. 🤣

Comment (AngelaFabregues) [reply helix-query.yaml:81]: In other words, thanks for the suggestion but please no. 😉

Comment (jvaldas) [reply helix-query.yaml:81]: @nc-andreashaller, I prefer to have a separate language index under /de/query-index-search.json. DE is out of MVP scope anyway. @AngelaFabregues, I can update the wiki if needed.

**Review by jvaldas — APPROVED** [review]

**Pull request merged**